### PR TITLE
Update using eclipse w/ link to beta plugin

### PR DESCRIPTION
**Pull request by jonl-percsolutions-com**
Added a link to the open source GWT Eclipse Plugin beta.

---
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/221)

### CODE REVIEW

File: src/main/markdown/usingeclipse.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ GWT release candidates are not bundled with The Google Plugin For Eclipse. If yo
   style="width: 385px; height: 100px;" alt="screenshot"/>
 </div>
 
+A beta version of the community GWT Plugin for Eclipse is available [here](https://gwt-plugins.github.io/documentation/gwt-eclipse-plugin/Download.html).  
+
+**DO NOT** attempt to use both versions of the plugin together.
```
Comment (tbroyer) [inline src/main/markdown/usingeclipse.md]: See also #218. Maybe we should drop all mention of the Google Plugin for Eclipse ?

Comment (jonl-percsolutions-com) [inline src/main/markdown/usingeclipse.md]: Seems that the Google Plugin was not updated for Neon, so I think you are right, we should just drop mention of the old Google plugin.

Comment (branflake2267) [inline src/main/markdown/usingeclipse.md]: GPE is on the way out. GCP or Google Cloud Tools for Eclipse is taking it's place.

**Comment by manolo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/main/markdown/usingeclipse.md, line 25 at r2](https://reviewable.io:443/reviews/gwtproject/gwt-site/221#-KiUfuGDm6l4lGlTeRUF:-KiUfuGDm6l4lGlTeRUG:bq8bnyu) ([raw file](https://github.com/gwtproject/gwt-site/blob/a9b471d50e458d09c86b978b12358ec90c9b1128/src/main/markdown/usingeclipse.md#L25)):*
> following 

GWT Plugin is now available in the Eclipse Marketplace, I think it's the easiest way to install it, can you make a mention to it ? 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/gwtproject/gwt-site/221)*

**Comment by zbynek** [comment]
This can be closed since https://github.com/gwtproject/gwt-site/pull/338 updated the plugin references.

**Comment by FrankHossfeld** [comment]
Agree, we should close this issue as fixed.

**Pull request closed without merging**